### PR TITLE
Add knmstate job to publish after master merge

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

File: github/ci/prow/files/jobs/nmstate/kubernetes-nmstate-postsubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+postsubmits:
+  kubevirt/project-infra:
```
Comment (phoracek) [inline github/ci/prow/files/jobs/nmstate/kubernetes-nmstate-postsubmits.yaml]: Is this correct?

Comment (qinqon) [reply github/ci/prow/files/jobs/nmstate/kubernetes-nmstate-postsubmits.yaml]: Nope, fixed it

**Comment by qinqon** [comment]
/hold

Looks like there is already a knmstate quay.io, we may need to use that instead of kubevirtbot.

**Comment by qinqon** [comment]
/hold cancel

kubevirtbot has being added to nmstate org so we good.

**Pull request merged**